tests/integration/tier0/inhibit-if-oracle-system-uses-not-standard-kernel/test_oracle_bad_kernel.py

```diff
@@ -9,21 +9,6 @@
 from envparse import env
 
 
-@pytest.mark.good_tests
```

Comment:
Why do you remove this case?

Reply:
The reason is that the "good" case does the conversion with suitable kernel and nothing else. It's the same test case as the basic ones in the `tier1`.